tests/components/spotify/test_config_flow.py

```diff
@@ -124,15 +108,12 @@ async def test_full_flow(
 
     with (
         patch("homeassistant.components.spotify.async_setup_entry", return_value=True),
-        patch("homeassistant.components.spotify.config_flow.Spotify") as spotify_mock,
     ):
-        spotify_mock.return_value.current_user.return_value = {
-            "id": "fake_id",
-            "display_name": "frenck",
-        }
         result = await hass.config_entries.flow.async_configure(result["flow_id"])
 
-    assert result["data"]["auth_implementation"] == "cred"
```

Comment:
Why is this changed?

Reply:
Because the auth implementation is now always the domain. I am not sure where `cred` comes from in a normal system